src/components/Rectangle/index.jsx

```diff
@@ -10,12 +10,13 @@ export default class Rectangle extends PureComponent {
   static propTypes = {
     title: PropTypes.string,
     description: PropTypes.string,
-    apps: PropTypes.array,
+    apps: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    total: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
`total` can be restricted to `number` type

Reply:
not load apps data, number defalut  '', not 0.